packages/bits/demo/.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+    "extends": "../.eslintrc.json",
```

Comment:
it seems like some properties in this file can be omitted since they have the same values as the one its extending

Reply:
I think we can omit **"createDefaultProgram": true**, but the rest should be there, the "overrides" key requires the definition of "files" types